### ISSUE TITLE
[v4] line-clamp overflow typo bug

### ISSUE THREAD

**Issue by hmbanan666** [tailwindlabs/tailwindcss#13217]
**What version of Tailwind CSS are you using?**

4.0.0-alpha.8

**What build tool (or framework if it abstracts the build tool) are you using?**

next 14.1.3, @tailwindcss/postcss 4.0.0-alpha.8

**What version of Node.js are you using?**

v20.11.1

**What browser are you using?**

Arc

**What operating system are you using?**

macOS

**Describe your issue**

line-clamp on output is broken:

```
.line-clamp-1 {
    overlow: hidden; /* Problem */
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-line-clamp: 1;
  }
  .line-clamp-2 {
    overlow: hidden;
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-line-clamp: 2;
  }
  .line-clamp-5 {
    overlow: hidden;
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-line-clamp: 5;
  }
```

Some fix needed on "overflow: hidden;", not "overlow: hidden;"

Thanks! v4 is awesome!

**Comment by hmbanan666** [comment]
Found this problem in:

```
packages/tailwindcss/src/utilities.ts on lines 789, 798
packages/tailwindcss/src/utilities.test.ts on lines 1328, 1335, 1342, 1349
```

**Comment by RobinMalfait** [comment]
Hey!

Thanks for the bug report. We've merged your PR (#13220) and will get a release out soon!
